### PR TITLE
Chronograf 1.6 Log Viewer

### PR DESCRIPTION
**Pull request by sanderson**

### CODE REVIEW

File: content/chronograf/v1.6/guides/analyzing-logs.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  chronograf_1_6:
+    weight: 120
+    parent: Guides
+---
+
+Chronograf gives you the ability to view, search, filter, visualize, and analyze log information from a variety of sources.
+This helps to recognize and diagnose patterns, then quickly dive into logged events that lead up to events.
+
+## Logging setup
+Logs data is a first glass citizen in InfluxDB and is populated using available log-related [Telegraf input plugins](/telegraf/latest/plugins/inputs/):
```
Comment (stevebang) [inline content/chronograf/v1.6/guides/analyzing-logs.md]: "first glass citizen" -- probably meant "first class citizen".

File: content/chronograf/v1.6/guides/analyzing-logs.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+![Searching and filtering logs](/img/chronograf/v1.6/logs-search-filter.gif)
+
+> **Note:** The log search field is case-sensitive.
+
+To remove filters, click the `×` next to the tag key by which you no longer want to filter.
+
+### Selecting specific times
+In the log viewer, you can select time ranges from which to view logs.
+By default, logs are stream and displayed relative to "now," but it is possible to view logs from a past window of time.
```
Comment (stevebang) [inline content/chronograf/v1.6/guides/analyzing-logs.md]: "logs are stream"?

File: content/chronograf/v1.6/guides/analyzing-logs.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+---
+
+Chronograf gives you the ability to view, search, filter, visualize, and analyze log information from a variety of sources.
+This helps to recognize and diagnose patterns, then quickly dive into logged events that lead up to events.
+
+## Logging setup
+Logs data is a first glass citizen in InfluxDB and is populated using available log-related [Telegraf input plugins](/telegraf/latest/plugins/inputs/):
+
+[logparser](https://github.com/influxdata/telegraf/tree/release-1.7/plugins/inputs/logparser)  
+[syslog](https://github.com/influxdata/telegraf/tree/release-1.7/plugins/inputs/syslog)
```
Comment (russorat) [inline content/chronograf/v1.6/guides/analyzing-logs.md]: the first version of the chronograf log viewer is built and tested to work with the syslog plugin.

File: content/chronograf/v1.6/guides/analyzing-logs.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+<img src="/img/chronograf/v1.6/logs-nav-log-viewer.png" alt="Log viewer in the left nav" style="width:100%;max-width:209px;"/>
+
+The log viewer provides a detailed histogram showing the time-based distribution of log entries color-coded by log severity.
+It also includes a live stream of logs that can be searched, filtered, and paused to analyze specific time ranges.
+
+<img src="/img/chronograf/v1.6/logs-log-viewer.png" alt="Chronograf log viewer" style="width:100%;max-width:1016px;"/>
+
+### Searching and filtering logs
+Logs are searched using keywords or regular expressions.
+They can also be filtered by severity and any tag values included with the log entry.
```
Comment (russorat) [inline content/chronograf/v1.6/guides/analyzing-logs.md]: filtered by clicking a value in the table, such as severity or facility

File: content/chronograf/v1.6/guides/analyzing-logs.md (hunk 3 above)
Comment (sanderson) [reply content/chronograf/v1.6/guides/analyzing-logs.md]: So should only `syslog` be mentioned here?

**Pull request merged**